src/local-development/clone.ts

```diff
@@ -56,9 +52,17 @@ async function cloneAppToWorkspace(context: App): Promise<void> {
 		baseUrl: 'https://' + environment.url,
 		appId: context.name,
 		appVersion: context.version,
+		idMapping: {
```

Comment:
btw in backend:
webhhook = hook
connection = account

should it be on par?

Reply:
Current naming is correct here.

Explanation:

This `idMapping` property is the part of `makecomapp.json` file, which is faced to a user/developer. So the content of the file is considered as "frontend". It is the reason why I am explicitly renaming it on many places to this naming "webhook"/"connection", which is known to user/developer from our product frontend and from documentation.

I am trying to have no place where to talk about `account` or about `hook`, because it is only our backend naming (that comes from historical sourcecode+DB consequences).